### ISSUE TITLE
TTS RHVoice and Coqui (and maybe others) fails to run speech engine for texts with new lines.

### ISSUE THREAD

**Issue by Podiumnoche** [mkiol/dsnote#28]
`Flatpak` `4.1.0`
For now i tested only following engines:
- Coqui (MMS, Mai VITS)
- Espeak (MBROLA, Robot)
- Piper
- RHVoice

Espeak and Piper works for every text so far. Coqui and RHVoice can't read text if there's at least one new line.

Cause is probably that for newline it's creating empty task.
```
[D] 20:14:48.26 0x7fc8df77ed80 encode_speech:174 - task: SENTENCE_BEFORE_NEW_LINE
[D] 20:14:48.26 0x7fc8df77ed80 encode_speech:174 - task: 
[D] 20:14:48.26 0x7fc8df77ed80 encode_speech:174 - task: SENTENCE_AFTER_NEW_LINE
```
```
[E] 20:14:59.438 0x7fc8d09ff600 operator():260 - py error: ValueError: You need to define either `text` (for sythesis) or a `reference_wav` (for voice conversion) to use the Coqui TTS API.
```

**Comment by mkiol** [comment]
Thanks for the report. 

I've already fix this problem in 04c5269a890805fd9278bf8b4765ae27d78094a3. 

I'm planing to push new release in the next week. 

BTW, If you want to test upcoming version 4.2.0 right now, it is already available in 'flatpak-beta' repository. Here is the instruction how to enable it: https://discourse.flathub.org/t/how-to-use-flathub-beta/2111.

**Comment by mkiol** [comment]
Fixed in version 4.2.0.
